contrib/mesos/pkg/minion/server.go

```diff
@@ -204,6 +205,15 @@ func (ms *MinionServer) launchHyperkubeServer(server string, args *[]string, log
 		log.Infof("wrote %d bytes to %v", written, logFileName)
 	}()
 
+	// use given environment, but add /usr/sbin to the path for the iptables binary used in kube-proxy
```

Comment:
I don't like the proposed magic combining the actual and given paths. The path is pretty host-dependent. If there is a need to change it, the user must know what he is doing. I implemented it as "path-override" for that reason, plain and explicit.